py-polars/tests/test_series.py

```diff
@@ -666,6 +666,18 @@ def test_diff_dispatch() -> None:
     )
 
 
+def test_pct_change_dispatch() -> None:
+    s = pl.Series("a", [1, 2, 4, 8, 16, 32, 64])
+    expected = pl.Series("a", [None, None, float("inf"), 3.0, 3.0, 3.0, 3.0])
+    testing.assert_series_equal(s.pct_change(2), expected)
+
+    df = pl.DataFrame([s])
+    testing.assert_series_equal(
+        df.select(pl.col("a").pct_change())["a"],
+        pl.Series("a", [None, float("inf"), 1.0, 1.0, 1.0, 1.0, 1.0]),
+    )
```

Comment:
You can use the function `verify_series_and_expr_api` to do these two tests in one go. I.e.

```python
s = pl.Series("a", [1, 2, 4, 8, 16, 32, 64])
expected = pl.Series("a", [None, None, float("inf"), 3.0, 3.0, 3.0, 3.0])
verify_series_and_expr_api(s, expected, "pct_change", 2)
```

As we have a lot of methods that apply to both Series and Dataframe+Expr.